Frontend-CB/src/assets/Navbar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Navbar: React.FC = () => {
+  return(
+    <nav>
```

Comment:
Also please add styling to this navbar. To let you know we are using tailwindcss.